calc/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "calc",
+  "version": "0.3.0",
+  "description": "Logic behind the official damage calculator for the Pokémon Showdown battle simulator",
+  "homepage": "http://pokemonshowdown.com/calc",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Zarel/honko-damagecalc.git"
+  },
+  "author": {
+    "name": "Guangcong Luo",
+    "email": "zarel@zarel.com",
+    "url": "http://guangcongluo.com"
+  },
+  "contributors": [
+    {
+      "name": "Mathieu Dias-Martins",
+      "email": "marty@hisemail.com"
```

Comment:
We can remove this, i don’t think Marty cares enough about the calc to want to be notified about Travis, etc

Reply:
Also it’s obviously not a real email, I was going to ask him about adding it but figured it wouldn’t matter